include/boost/geometry/util/rational.hpp

```diff
@@ -16,6 +16,7 @@
 
 #include <boost/rational.hpp>
 #include <boost/numeric/conversion/bounds.hpp>
+#include <boost/numeric/conversion/converter.hpp>
```

Comment:
So indeed this fixed it.
I fixed it in another way (again, because I didn't notice this earlier, sorry) and I think it is a better fix. See #1246